### DISCUSSION TITLE
Requesting a timed option to stop showing current results and show the next heat

### DISCUSSION THREAD

**Discussion by kemton** [JohnStrunk/wahoo-results#502] [Ideas & feature requests]
I have tested wahoo at home and used it for the first time last night at an official meet. It worked very well. Thank you for building it. The swimmers/heats are defined in the exported scb files. I would like an option to replace the results displayed with the swimmer names in the next heat, either with the press of a button or automatically after a configurable number of seconds. The results of the last heat remain on the scoreboard during the next heat until those swimmers are finished. 

![scoreboard](https://github.com/JohnStrunk/wahoo-results/assets/54498835/532dec00-46f3-46c4-9b41-bf71cb386135)

**Comment by JohnStrunk** [comment]
@kemton, I like that idea; thanks for suggesting it. I'm going to have to give some thought to what the interface would look like.
It's similar to a feature that I had in the pre-1.0 versions where you could re-display previous results, but the interface to that feature was horrible.

The auto-advance is a great touch-- I've tried to keep the scoreboard software such that once it's set up for the session, it can just be ignored by the AOs and timing system operator, and auto-advance keeps with that theme.

Also, thank you for the picture! It's great to see folks using the scoreboard.
